py3-cassandra-medusa.yaml

```diff
@@ -2,13 +2,16 @@
 package:
   name: py3-cassandra-medusa
   version: 0.19.1
-  epoch: 1
+  epoch: 2
   description: Apache Cassandra backup and restore tool
   copyright:
     - license: Apache-2.0
   options:
     no-provides: true
     no-depends: true
+  dependencies:
+    runtime:
+      - python-3.11-base
```

Comment:
does this need to be tied to python 3.11?

Reply:
Oh nice. Will close this one then. Thank you so much for your help with this!

Reply:
Oh wait. Not this one. The other one. My had